src/components/Header.js

```diff
@@ -8,11 +8,11 @@ const Header = () => {
   <NavItem>
     <a href="/catindex">See All The Cats</a>
   </NavItem>
-  <NavItem>
+  {/* <NavItem>
```

Comment:
I think these lines want to get deleted. They won't be useful in the future.

Reply:
Resolved!